themes/default/content/blog/pulumiup/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "PulumiUP - the Event for Cloud Engineers"
+date: 2021-04-14
+meta_desc: "PulumiUP is two hour keynote with cloud engineering industry leaders Adrian Cockcroft, VP of Sustainability Engineering at Amazon, Justin Fitzhugh, VP of Cloud Engineering at Snowflake, Kieth Redmond, VP of SaasS Engineering at Fenergo."
+meta_image: pulumiup.png
+authors:
+    - wendy-smith
+tags:
+    - cloud engineering
+    - PulumiUP
+---
+
+My professional background has included nearly ten years of managing field events and user conferences. I never thought I would say this, but I miss traveling. I even missed Vegas and reInvent this year. I miss connecting with customers and advocates in our communities. I wish we could all be looking forward to getting together in person in Seattle or Austin or insert any city here. As the year continued, it became clear we were not going back to in-person events anytime soon, and everyone in the industry pivoted to virtual programs while video conferencing became an all-day activity.
+
+<!--more-->
+
+We spent a lot of time considering how to best engage and share announcements with our community and encourage the cloud engineering community's growth. All of us here at Pulumi recognize how valuable your time is and how much time in the last year we spent on virtual event platforms. With that in mind, we made a deliberate decision to avoid any all-day or multi-day programming. Instead, we are hosting a short 2-hour keynote that we think you will find insightful and valuable - best practices and real-life situations that you can learn from and apply to your approach to building, delivering, and managing modern applications.
```

Comment:
```suggestion
We spent a lot of time considering how to best engage and share announcements with our community and encourage the cloud engineering community's growth. All of us here at Pulumi recognize how valuable your time is and how much time we've all spent on virtual event platforms in the last year. With that in mind, we made a deliberate decision to avoid any all-day or multi-day programming. Instead, we are hosting a short 2-hour keynote that we think you will find insightful and valuable, with best practices and real-life situations that you can learn from and apply to your approach to building, delivering, and managing modern cloud applications.
```